k8s/cmd/admission-webhook/patches.go

```diff
@@ -19,54 +19,60 @@ type patchOperation struct {
 }
 
 func createDNSPatch(tuple *podSpecAndMeta, annotationValue string) (patch []patchOperation) {
+	// TODO: now order of containers is important since nsmdp assign proper workspace only to the first container
 	patch = append(patch, addContainer(tuple.spec,
 		[]corev1.Container{
 			{
-				Name:            "nsm-coredns",
-				Image:           fmt.Sprintf("%s/%s:%s", getRepo(), "nsm-coredns", getTag()),
+				Name:            "nsm-dns-monitor",
```

Comment:
Why is the security patch changing the DNS feature?

Reply:
I just swapped containers, as I already mentioned only the first container receives the correct workspace